src/radix_connect/wallet_interaction/dapp_wallet_interaction/wallet_to_dapp/success_response/auth/auth_proof.rs

```diff
@@ -0,0 +1,27 @@
+use crate::prelude::*;
+
+#[derive(Debug, Serialize, PartialEq, uniffi::Record)]
+#[serde(rename_all = "camelCase")]
+pub struct WalletToDappInteractionAuthProof {
+    pub public_key: String,
+    pub curve: SLIP10Curve,
+    pub signature: String,
```

Comment:
we should probably take the opportunity to use stronger types for `public_key` and `signature` than using `String`, you should be able to use `PublicKey` and `Signature` directly I think. 

Especially since this is Wallt TO Dapp - the wallet will know how to produce valid values. So most def should be `PublicKey` and `Signature`